package.json

```diff
@@ -62,8 +61,11 @@
   "lavamoat": {
     "allowScripts": {
       "@lavamoat/preinstall-always-fail": false,
-      "keccak": true,
-      "secp256k1": true
+      "eth-sig-util>ethereumjs-util>keccak": false,
```

Comment:
These were all `true` before, but seemingly they aren't needed at all for local development, so I left them disabled.